highfive/configs/rust-lang/rust-guide.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Should this file be rustc-guide.json (note the "c" in rustc)?

Reply:
Yep, woops, already pushed a fix.